packages/dx-react-scheduler/docs/guides/fundamentals.md

```diff
@@ -23,7 +23,7 @@ To specify the date that should be initially displayed in the date navigator and
 
 .embedded-demo({ "path": "scheduler-basic/basic-setup", "showThemeSelector": true })
 
-## Appearance Customization
+## Customize the Appearance
```

Comment:
We very often add this link to our customers in issues. Can we add a redirect to this new ref?

Reply:
But in the Fundamentals section, we have only one big subsection) In this case redirect is not need

Reply:
AFAIK, there's no way to redirect from an anchor link, so I'm afraid we can't do this